apps/apps.humanatlas.io/src/app/components/server-selector/server-selector.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ServerSelectorComponent } from './server-selector.component';
+import { servers } from '../../constants/server.constants';
+
+describe('ServerSelectorComponent', () => {
+  let component: ServerSelectorComponent;
+  let fixture: ComponentFixture<ServerSelectorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
```

Comment:
Please update this to use testing-library to render the component instead of the default generated code